public/css/style.css

```diff
@@ -183,4 +183,4 @@ div.copyright {
 
 span#copy {
```

Comment:
avoid giving CSS for ids. It's better to use classes

Reply:
yeah, again I did not do this one

Reply:
I think we need to fix the basic css first. Wdyt?

Reply:
Yeah well! 
i think we should have a proper team meet with at-lease regular contributors once to see the best rout and CSS fixes